gradle/libs.versions.toml

```diff
@@ -34,8 +33,8 @@ micrometer-registry-otlp = { module = "io.micrometer:micrometer-registry-otlp" }
 micrometer-tracing-bridge-brave = { module = "io.micrometer:micrometer-tracing-bridge-brave" }
 spring-boot-starter-actuator = { module = "org.springframework.boot:spring-boot-starter-actuator" }
 spring-boot-starter-jdbc = { module = "org.springframework.boot:spring-boot-starter-jdbc" }
-spring-boot-starter-undertow = { module = "org.springframework.boot:spring-boot-starter-undertow" }
```

Comment:
`undertow` does not support Spring Boot 4 yet

https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-4.0-Migration-Guide

<img width="916" height="155" alt="Image" src="https://github.com/user-attachments/assets/90afe762-8ba4-40a5-a88c-df3c9e2487ed" />